_blogposts/2024-02-01-release-11-1-0.mdx

```diff
@@ -0,0 +1,257 @@
+---
+author: rescript-team
+date: "2024-03-04"
+previewImg: /static/blog/compiler_release_11_1.jpg
+title: ReScript 11.1
+badge: release
+description: |
+  Unleashing ReScript from React
+---
+
+A couple of weeks ago, the ReScript team [released ReScript 11.0](/blog/release-11-0-0), which laid ground work for a lot of possible improvements to make it easier to interact with the JavaScript ecosystem.
```

Comment:
This is no longer a couple of weeks ago 😄

Reply:
better now?